packages/h5p-express/src/ContentUserDataRouter/ContentUserDataController.ts

```diff
@@ -0,0 +1,59 @@
+import * as express from 'express';
+
+import {
+    ContentUserDataManager,
+    IPostContentUserData
+} from '@lumieducation/h5p-server';
+import { IRequestWithUser } from '../expressTypes';
+
+interface IPostContentUserDataRequest
+    extends IPostContentUserData,
+        IRequestWithUser {}
+
+export default class ContentUserDataController {
+    constructor(protected contentUserDataManager: ContentUserDataManager) {}
+
+    /**
+     * Returns the userState for given contentId, dataType and subContentId
+     */
+    public getContentUserData = async (
+        req: IRequestWithUser,
+        res: express.Response<{
+            data: string;
+            success: boolean;
+        }>
+    ): Promise<void> => {
+        const { contentId, dataType, subContentId } = req.params;
+        const userState = await this.contentUserDataManager.loadContentUserData(
+            contentId,
+            dataType,
+            subContentId,
+            req.user
+        );
+        res.status(200).json({ data: userState, success: true });
+    };
+
+    /**
+     * Saving a userState for given contentId, datType and subContentId
+     *
+     */
+    public postContentUserData = async (
+        req: IPostContentUserDataRequest,
+        res: express.Response
+    ): Promise<void> => {
+        const { contentId, dataType, subContentId } = req.params;
```

Comment:
potentially unsafe parameters must be validated / sanitized

Reply:
I will sanitize these things in the ContentUserDataManager as there will be no need for the user to do so if the user decides to handle the implementation on their own.